### DISCUSSION TITLE
hide or leave hand of the initial animation

### DISCUSSION THREAD

**Discussion by TheDrake94** [google/model-viewer#3305] [General]
Hi, everyone! I wanted to ask if it was possible to remove the initial animation and that the rotation starts automatically, or that it makes its animation "jerky" but without seeing the hand in front of the object, thanks to all.

**Comment by elalish** [comment]
You can control the interaction prompt with several [attributes](https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-attributes-interactionPrompt), and you can replace the hand with our [`<slot>`](https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-slots-interactionPrompt)

**Reply by TheDrake94** [reply]
Thank you so much ! it's work!
